bindata/network/infra-dpu-cni/infra-dpu-cni.yaml

```diff
@@ -0,0 +1,163 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: infra-dpu-cni-script
```

Comment:
There's nothing DPU-specific about the script itself...

Reply:
Nothing DPU specific other than it is intended to run only on the DPU. There is a [noop CNI](https://github.com/containernetworking/cni/tree/master/plugins/test/noop), so didn't want to confuse with that one. Is there a better name?